src/main/java/com/greenfoxacademy/goddesstribesbackend/models/dtos/KingdomDTO.java

```diff
@@ -2,19 +2,19 @@
 
 public class KingdomDTO {
 
-  private Long id;
+  private Long kingdomId;
   private String kingdomname;
   private String username;
 
   public KingdomDTO() {
   }
 
-  public Long getId() {
-    return id;
+  public Long getKingdomId() {
+    return kingdomId;
   }
```

Comment:
Like this way, when you have all the setters and getters, you will have something like
`myKinddom.getKingdomId()` which is redundant, instead of having `myKingdom.getId()`